source/App.h

```diff
@@ -98,8 +98,11 @@ class App : public GApp {
 	Array<shared_ptr<FloatingCombatText>>	m_combatTextList;					///< Array of existing combat text
 
 	shared_ptr<Weapon>						m_weapon;							///< Current weapon
-	shared_ptr<ArticulatedModel>			m_decalModel;						///< Model for the miss decal
-	Array<shared_ptr<VisibleEntity>>		m_currentDecals;					///< Storage for miss decals
+	shared_ptr<ArticulatedModel>			m_missDecalModel;					///< Model for the miss decal
+	shared_ptr<ArticulatedModel>			m_hitDecalModel;					///< Model for the hit decal
+	shared_ptr<VisibleEntity>				m_hitDecal;							///< Pointer to hit decal
```

Comment:
Spacing looks weird here. Check if tabs are doing something wrong...

Reply:
Hmm, odd this looks correct in VS. It might also be time to make a tabs vs spaces determination for FPSci.

Reply:
Given that G3D uses "spaces as tabs" we probably want to use spaces everywhere. Not sure how most repositories add style guides, but we should include something with the repo.

Reply:
Okay, I think this belongs in a separate commit/pull request. I'd suggest we leave this as-is and come back around to it in another item. 

Do we want to create a github issue to track converting to spaces? Should we do anything to enforce this on check-in (other than use of pull requests)?

Reply:
Yeah, the full convert to spaces should be a separate issue/pull request. I'm okay with adding an "action"